databases.go

```diff
@@ -2,11 +2,20 @@ package scalingo
 
 import (
 	"context"
+	"net/http"
 	"time"
 
 	"gopkg.in/errgo.v1"
+
+	httpclient "github.com/Scalingo/go-scalingo/v5/http"
 )
 
+type DatabasesService interface {
```

Comment:
Shouldn't you also create a mock for the `DatabasesService`? I think we create it for all services.